### ISSUE TITLE
Maintenance in Microsoft Edge Flatpak package

### ISSUE THREAD

**Issue by TheEvilSkeleton** [MicrosoftEdge/Status#704]
Hi,

My apologies if I have opened this issue in the wrong repository. I have contacted Microsoft via the forums a while ago, but I have not gotten a response.

A couple of months ago, I have submitted a Microsoft Edge Flatpak package in Flathub and have maintained the package since then, see https://github.com/flathub/flathub/pull/1992.

In short, Flatpak is a universal package manager for Linux. While the .deb and .rpm is available which are the two most used package formats in Linux, Flatpak sandboxes all applications for security purposes, and we have managed to nest Edge's sandbox. Just to be clear, **we do not redistribute Microsoft Edge!** We are using the [`extra-data`](https://github.com/flathub/com.microsoft.Edge/blob/4a8c235f75dba9021c0fa99c3af7d631cad7a231/com.microsoft.Edge.yaml#L102) parameter to avoid redistributing binaries, more details in this [YouTube video](https://www.youtube.com/watch?v=xnnJRP4t9gM&t=3220s).

Since the Flatpak package is an unofficial package, as in it is not maintained by the Edge team, I would like to ask if the Edge team can maintain the Flatpak package so the user can directly interact with the team in the issue tracker. To do so, you will have to contact the Flathub maintainers. The instructions are available [here](https://github.com/flathub/flathub/wiki/App-Submission#someone-else-has-put-my-app-on-flathubwhat-do-i-do).

Currently, since only the Dev and Beta builds are available on Linux, the Flatpak package only ships the Beta builds in the beta branch of Flathub. Once Edge Stable is available on Linux *and* the Chrome Flatpak package gets to the stable branch (it is not as we speak), we will be safe to maintain the Stable builds in the stable branch, and Beta builds in the beta branch.

Only Edge Beta is available in Flathub. I wouldn't mind submitting Edge Dev in Flathub (ID is going to be `com.microsoft.Edge.Dev` in the Flathub beta branch) to have both Beta and Dev in Flathub.

Thank you.

**Comment by travisleithead** [comment]
Thanks for reaching out. My intuition is that the Edge team chose their Linux distribution strategy with care and would **not** be interested in maintaining an addtional Flatpak package. However, adding @TheLarkInn for a more official comment from someone closer to the Linux plans.

**Comment by TheEvilSkeleton** [comment]
That's honestly disappointing.

**Comment by meisme-dev** [comment]
> Thanks for reaching out. My intuition is that the Edge team chose their Linux distribution strategy with care and would **not** be interested in maintaining an addtional Flatpak package. However, adding @TheLarkInn for a more official comment from someone closer to the Linux plans.

flatpak would be a better choice due to the extra sandboxing and distrubution support + auto updates.

**Comment by castrojo** [comment]
They've published instructions on how to enable edge on the deck using the beta flatpak so I reached out to them to see if they want to work together in some capacity.

**Comment by GitThisBugOff** [comment]
Seeing how Microsoft now "officially" supports the Steam Deck for Xbox Cloud, the Edge team should reconsider their stance on this topic.

**Comment by m00nwtchr** [comment]
Please, no
https://ludocode.com/blog/flatpak-is-not-the-future

**Comment by TheEvilSkeleton** [comment]
> Please, no https://ludocode.com/blog/flatpak-is-not-the-future

Counter blog post: https://blogs.gnome.org/wjjt/2021/11/24/on-flatpak-disk-usage-and-deduplication/

**Comment by m00nwtchr** [comment]
> > Please, no https://ludocode.com/blog/flatpak-is-not-the-future
> 
> Counter blog post: https://blogs.gnome.org/wjjt/2021/11/24/on-flatpak-disk-usage-and-deduplication/

Which only addresses one of the issues brought up in the other post (and still gives pretty bad results, almost 9GB of redundant data), and there is also https://flatkill.org/

**Comment by o-kotb** [comment]
Yes, they should officially maintain a flatpak package by either working with the current maintainers or make their own.

**Comment by TheEvilSkeleton** [comment]
Just an update for everyone. I recently published an article[1] about an oversight by Microsoft towards the Microsoft Edge Flatpak app. I expressed my concerns and have also requested them to officially maintain the Flatpak. I got an email from the Product Manager a couple of hours later, stating that they are working to officially release Edge as Flatpak. Hopefully this happens.

1: https://theevilskeleton.gitlab.io/2022/03/19/a-letter-to-microsoft-for-not-attributing-authors-of-the-edge-flatpak-application.html

**Comment by GeorgesStavracas** [comment]
Thanks @TheEvilSkeleton, that sounds promising! If there's anything needed from the portals front, I'd love to help in any capacity, just let me know.

**Comment by Taulim** [comment]
Just want to point out my frustration with Microsoft's shameful attitude towards the flatpak maintainers.
Disappointed, but not surprised.

First they ignored flatpak silently for a whole year and suddenly it become interesting.
But worst part was not even mentioning the people that made it easier for them.
Lets be fair, the official post got indeed update. (👏 Thank!)
I understand that maybe it was not intentional / maliciously, everyone makes mistakes (or people that wanted to make it right did not have the 'power').
I also know as it was not the first time and will not be the last.

As far as I understand it is in the Edge team (and Microsoft) interest that it succeed as flatpak and more people use the software.

I really hope that they officially help to support flatpak and we get a better experience.
Edge has some good features! But I don't like to give any browser access to everything :)

**Comment by Wladefant** [comment]
@TheEvilSkeleton are there any updates?

**Comment by Wladefant** [comment]
@TheEvilSkeleton any updates?

**Comment by TheEvilSkeleton** [comment]
I contacted them and have not received a response yet
